File: angular/src/app/services/bus-route.service.ts
import { Injectable } from "@angular/core";
import { BusRouteStore } from "../stores/bus-route-store.service";
import { BusRoute } from "../models/bus-route";
import { CreateBusRouteCommand } from "../commands/createBusRoute";

@Injectable()
export class BusRouteService {

  constructor(private busRouteStore: BusRouteStore) {

  }


  find(): Promise<BusRoute[]> {
    return this.busRouteStore.find();
  }


  create(cmd: CreateBusRouteCommand) : Promise<BusRoute> {
    return this.busRouteStore.create(cmd);
  }
  
}

File: server/TicketApp.Service/BusRouteService.cs
using System.Collections.Generic;
using TicketApp.Domain.Command;
using TicketApp.Domain.Model;
using TicketApp.Repository;

namespace TicketApp.Service
{
    public class BusRouteService
    {
        private readonly BusRouteRepository _busRouteRepository;


        public BusRouteService(BusRouteRepository busRouteRepository)
        {
            _busRouteRepository = busRouteRepository;
        }


        public IEnumerable<BusRoute> Find()
        {
            return _busRouteRepository.Find();
        }


        public BusRoute Create(CreateBusRouteCommand cmd)
        {
            return _busRouteRepository.Create(cmd);
        }

    }
}
